dataactvalidator/validation_handlers/validator.py

```diff
@@ -782,3 +789,27 @@ def validateTAS(fieldsToCheck, tasFields, record, interfaces, fileType):
         else:
             # Multiple instances of same TAS, something is going wrong
             raise ResponseException("TAS check is malfunctioning",StatusCode.INTERNAL_ERROR)
+
+    @staticmethod
+    def validateFileBySql(submissionId,fileType,interfaces):
+        # Pull all SQL rules for this file type
+        fileId = interfaces.validationDb.getFileId(fileType)
+        rules = interfaces.validationDb.session.query(RuleSql).filter(RuleSql.file_id == fileId).all()
+        errors = []
+        # For each rule, execute sql for rule
+        for rule in rules:
+            failures = interfaces.validationDb.connection.execute(rule.rule_sql.format(submissionId))
+            # Build error list
+            for failure in failures:
+                row = failure["row"]
+                errorMsg = rule.rule_description
```

Comment:
The sql rule table has a separate `rule_error_message` column that I had envisioned as the error message. This would be to handle cases where we've split a rule into multiple SQL statements (B11 for example...one error message would about an incorrect objectclass, and one message would be about an invalid objectclass prefix).

What do you think about that approach?